### PR TITLE
Limelight Digital Bid Adapter: added new custom fields for targeting

### PR DESCRIPTION
**Pull request by apykhteyev**
## Type of change
- [X] Feature
- [X] Does this change affect user-facing APIs or examples documented on http://prebid.org?

## Description of change

Added new custom fields for targeting on Limelight platform

## Other information
https://github.com/prebid/prebid.github.io/pull/4288

### CODE REVIEW

**Comment by apykhteyev** [comment]
@gwhigs doesn't seems that my tests were failed in the build

**Review by gwhigs — COMMENTED** [review]
I wasn't able to see the test failure in CircleCI, but it's possible a lint check failed on these.

File: modules/limelightDigitalBidAdapter.js [hunk 1]
```diff
@@ -55,8 +55,9 @@ export const spec = {
       winTop = window;
     }
     const placements = groupBy(validBidRequests.map(bidRequest => buildPlacement(bidRequest)), 'host')
-    return Object.keys(placements)
+    let map = Object.keys(placements)
       .map(host => buildRequest(winTop, host, placements[host].map(placement => placement.adUnit)));
+    return map;
```
Comment (gwhigs) [inline modules/limelightDigitalBidAdapter.js:60]: This change seems unnecessary. Is there a particular reason you included it?

File: modules/limelightDigitalBidAdapter.js [hunk 2]
```diff
@@ -113,7 +114,7 @@ export const spec = {
 
 registerBidder(spec);
 
-function buildRequest(winTop, host, adUnits) {
+function buildRequest(winTop, host, adUnits, bidRequest) {
```
Comment (gwhigs) [inline modules/limelightDigitalBidAdapter.js:117]: This argument is unused.

**Review by gwhigs — COMMENTED** [review]
I wasn't able to see the test failure in CircleCI, but it's possible a lint check failed on these.

File: modules/limelightDigitalBidAdapter.js (hunk 1 above)
Comment (apykhteyev) [reply modules/limelightDigitalBidAdapter.js]: fixed

File: modules/limelightDigitalBidAdapter.js (hunk 2 above)
Comment (apykhteyev) [reply modules/limelightDigitalBidAdapter.js]: fixed

**Comment by apykhteyev** [comment]
@gwhigs could you please check it?

**Comment by apykhteyev** [comment]
@gwhigs any updates?

**Pull request merged**